include/raylib-nuklear.h

```diff
@@ -496,9 +513,9 @@ DrawNuklear(struct nk_context * ctx)
                 image.mipmaps = 1;
                 image.format = PIXELFORMAT_UNCOMPRESSED_R8G8B8A8;
                 Texture texture = LoadTextureFromImage(image);
-                Rectangle source = (Rectangle){0, 0, (float)image.width, (float)image.height};
-                Rectangle dest = (Rectangle){(float)i->x, (float)i->y, (float)i->w, (float)i->h};
-                Vector2 origin = (Vector2){0, 0};
+                Rectangle source = {0, 0, (float)image.width, (float)image.height};
+                Rectangle dest = {(float)i->x, (float)i->y, (float)i->w, (float)i->h};
+                Vector2 origin = {0, 0};
```

Comment:
Just straight up removes the () before the declaration.

Reply:
Absolutely.